MiniApp/Classes/Environment.swift

```diff
@@ -54,7 +52,7 @@ internal class Environment {
     }
 
     var hostAppUserAgentInfo: String {
-        return value(for: customHostAppUserAgentInfo, fallback: .hostAppUserAgentInfo)
+        return value(for: .none, fallback: .hostAppUserAgentInfo)
```

Comment:
you could directly return hostAppUserAgentInfo from plist if you never intend to override it.

Reply:
Done